packages/near-membrane-dom/src/browser-realm.ts

```diff
@@ -1,21 +1,41 @@
 import {
+    init,
     VirtualEnvironment,
-    EnvironmentOptions,
-    ReflectApply,
-    ObjectLookupOwnGetter,
-    emptyArray,
-    linkIntrinsics,
     getFilteredEndowmentDescriptors,
-    setupStackTrace,
+    ProxyTarget,
 } from '@locker/near-membrane-base';
 
-import { getCachedBlueReferences, getRedReferences, linkUnforgeables, tameDOM } from './window';
+import { getCachedBlueReferences, getRedReferences, tameDOM } from './window';
 
+interface EnvironmentOptions {
+    distortionCallback?: (originalTarget: ProxyTarget) => ProxyTarget;
+    endowments?: object;
+    globalThis: WindowProxy & typeof globalThis;
+}
+
+const emptyArray: [] = [];
 const IFRAME_SANDBOX_ATTRIBUTE_VALUE = 'allow-same-origin allow-scripts';
+// TODO: how to guarantee that the function is actually running in strict mode?
+const initSourceText = `(function(){'use strict';return (${init.toString()})})()`;
```

Comment:
The TODO comment doesn't make sense as the `'use strict'` directive does guarantee it.

Reply:
well, I was asking for a better way of doing it :), I was hoping that you have a more elegant way of doing this.

Reply:
I have some changes in a testing followup that address this